clients/hadoopfs/pom.xml

```diff
@@ -52,10 +52,26 @@ To export to S3:
 
     <profiles>
         <profile>
-            <id>contract-tests</id>
+            <id>tests-hadoop2</id>
```

Comment:
I kinda _liked_ the word "contract" in there.  Unless it's incorrect, can we keep it please?